config/default.yaml

```diff
@@ -18,7 +27,7 @@ influxdb:
   password: 'root'
   use_udp: false
   retentionPolicy: 'default'
-  # Acceptable version are: '0.8' and '0.9'
+  # All version other than '0.9' will default to '0.8'
```

Comment:
This is kind of odd.  It should probably error if you try to enter a value other than '0.8' or '0.9'